scripts/fgi.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
```

Comment:
There's nothing wrong with bash, but I usually like to use python or some little more user friendly language so maintainability is easier.  Of course then you have to deal with python versions and it's less portable.

Reply:
yea, we have discussed about this point, since script does not need to compiling etc and it actually is simple enough by logic(yes it is a long script), maybe we may keep it in sh on MVP stage. PE team split this part for more feature works, so literally I pulled latest change from https://github.com/diem/diem/pull/8603#discussion_r659957795 😆